$NetBSD: patch-bb,v 1.1 2009/02/12 02:36:56 mwdavies Exp $

Emit the correct FilesRemoved signal if the job was aborted in the middle
of its operation, otherwise it can result in confusion and data loss
(overwriting files with files that don't exist).

svn 917170

--- branches/KDE/4.2/kdelibs/kio/kio/copyjob.h	2009/01/27 01:22:42	917169
+++ kio/kio/copyjob.h	2009/01/27 01:24:26	917170
@@ -245,6 +245,7 @@
 
     protected:
         CopyJob(CopyJobPrivate &dd);
+        void emitResult();
 
     private:
         Q_PRIVATE_SLOT(d_func(), void slotStart())
